mlflow/store/local_artifact_repo.py

```diff
@@ -10,13 +10,19 @@ class LocalArtifactRepository(ArtifactRepository):
     """Stores artifacts as files in a local directory."""
 
     def log_artifact(self, local_file, artifact_path=None):
+        if artifact_path and (artifact_path.startswith("/") or artifact_path.startswith(".")):
```

Comment:
Are we trying to disallow `..`? Theoretically, `.foo` is a valid file name. Maybe we could use this utility:
https://github.com/mlflow/mlflow/blob/e79920f44b3852675abd0c14d16b3fb9af44b7fd/mlflow/utils/validation.py#L27

Reply:
ooh! good one...